docs/kanvas-packages/workflows/rules.md

```diff
@@ -0,0 +1,124 @@
+# The Rule
+
+The rules are the main feature in this. The rule is compose for the Rule, the actions trigered and the conditions. Each Actions have the specifics System modules,
+For Example, PDF currently just work with the Messages system modules, no leads.
+
+## Before 
+Currently the Workflow no have the interface for create rule, here we going to create the rule in the database. You must have the system_modules_id of the entity 
```

Comment:
"Currently, Workflows does not have an interface for rule creation. Rules are created directly on the database and an entity's system module id must be provided".